_pages/business-units/18f/projects-partners/collaboration-tools.md

```diff
@@ -226,7 +214,7 @@ Here are some recommendations for how to level up our partners' skills:
 
 ## Using partners' tools
 
-Per the [IT Standards policy](https://www.gsa.gov/directive/gsa-information-technology-(it)-standards-profile), section 2(e):
+Per the [IT Standards policy](<https://www.gsa.gov/directive/gsa-information-technology-(it)-standards-profile>), section 2(e):
```

Comment:
Does adding angle brackets here work in markdown?

Reply:
Ah, yep. [Angle brackets can be used as delimiters.](https://meta.stackexchange.com/a/185066/159001) [Prettier](https://prettier.io/) automatically adds those when there are parentheses present in the URL.